action/recipes/edk2_test.go

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: MIT
+
+// Package recipes / edk2
+package recipes
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"dagger.io/dagger"
+	"github.com/9elements/firmware-action/action/container"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEdk2(t *testing.T) {
+	// This test is really slow (like 100 seconds)
+	if testing.Short() {
+		t.Skip("skipping test in short mode")
+	}
+
+	pwd, err := os.Getwd()
+	assert.NoError(t, err)
+	defer os.Chdir(pwd) // nolint:errcheck
+
+	// Use "" if you want to test containers from github package registry
+	// Use "../../docker/linux" if you want to test containers built fresh from Dockerfile
```

Comment:
This comment needs to updated to say `../../docker/linux` I guess

Reply:
ups :D